infra/gcp/infra.yaml

```diff
@@ -318,6 +318,7 @@ infra:
       k8s-staging-multitenancy:
       k8s-staging-networking:
       k8s-staging-nfd:
+      k8s-staging-nfs-server-provisioner:
```

Comment:
Thanks for digging into this! Could you double check if https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-sig-storage/generate.sh#L38 already covers the repo?

Reply:
Yeah, you're right, thanks for pointing to it

/close